browser/ui/views/side_panel/chat_ui/chat_ui_side_panel_coordinator.cc

```diff
@@ -0,0 +1,83 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/side_panel/chat_ui/chat_ui_side_panel_coordinator.h"
+
+#include "brave/browser/ui/webui/ai_chat/ai_chat_ui.h"
+#include "brave/components/constants/webui_url_constants.h"
+#include "chrome/browser/profiles/profile.h"
+#include "chrome/browser/ui/views/side_panel/side_panel_registry.h"
+#include "chrome/browser/ui/views/side_panel/side_panel_web_ui_view.h"
+#include "components/grit/brave_components_strings.h"
+#include "ui/base/l10n/l10n_util.h"
+
+namespace {
+class ChatUISidePanelWebView : public SidePanelWebUIView {
+ public:
```

Comment:
Having METADATA_HEADER for this view and its impl macros would be great

Reply:
Added